misc/docerina/src/main/java/org/ballerinalang/docgen/Generator.java

```diff
@@ -189,8 +183,8 @@ public static Page generatePageForPrimitives(BLangPackage balPackage, List<Link>
         for (Link primitiveType : primitives) {
             String type = primitiveType.caption.value;
             String desc = BallerinaDocUtils.getPrimitiveDescription(descriptions, type);
-            primitiveTypes.add(new PrimitiveTypeDoc(type, desc != null && !desc.isEmpty() ? BallerinaDocUtils
-                    .mdToHtml(desc) : desc, new ArrayList<>()));
+            primitiveTypes.add(new PrimitiveTypeDoc(type, !desc.isEmpty() ? BallerinaDocUtils.mdToHtml(desc) :
```

Comment:
The null check is removed. Are we certain that this string will not produce a null value?